### PR TITLE
[W5.10h][W13-A2] Aadyaa Maddi

### PR DESCRIPTION
**Pull request by amad-person**
Added more use cases for the AddressBook in the Developer Guide.

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -87,6 +87,42 @@ Use case ends.
...
+
+1. User requests to sort persons
+2. AddressBook shows a list of persons sorted in the alphabetical order of names.
+Use case ends.
+
+**Extensions**
+
+2a. The list is empty
+
+> Use case ends
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:103]: At least the system needs to inform the user that there are no people in the Addressbook before ending.

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -87,6 +87,42 @@ Use case ends.
...
+
+#### Use case: Rename tags
+
+**MSS**
+
+1. User requests to rename a particular tag
+2. AddressBook shows a list of persons with the tag
+3. AddressBook prompts user to enter new name for existing tag
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:111]: Use cases should depict the back and forth interaction between the user and the system. For every step the user does, the system should do something in return (and vice-versa).

File: doc/DeveloperGuide.md [hunk 3]
```diff
@@ -87,6 +87,42 @@ Use case ends.
...
+Use case ends.
+
+**Extensions**
+
+2a. There is no such tag 
+
+> 2a1. AddressBook shows an error message <br>
+  Use case resumes at step 1
+  
+4a. User cancels the rename operation
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:123]: - What happens when the new tag is already present in the address book?
- What happens if the new tag is same as the old tag?

**Comment by okkhoy** [comment]
@amad-person 
Some comments added. Please ack & close the PR after reading comments.

**Comment by amad-person** [comment]
Acknowledged, thanks

**Pull request closed without merging**